multinet/uploaders/nested_json.py

```diff
@@ -22,15 +22,14 @@ def analyze_nested_json(
     `data` - the text of a nested_json file
     `(nodes, edges)` - a node and edge table describing the tree.
     """
-    id = itertools.count(100)
+    _id = itertools.count(100)
```

Comment:
I don't think it's good to use a leading underscore like this, since that has a specific conventional meaning in Python code. Let's go with `ident` instead.

Reply:
Good point, I'll change it.